src/components/strategies/create/CreateForm.tsx

```diff
@@ -27,7 +27,8 @@ export const CreateForm: FC<FormProps> = (props) => {
   const { openModal } = useModal();
   const { user } = useWagmi();
   const search = useSearch({ strict: false }) as any;
-  const isSell = type === 'disposable' && search.direction === 'sell';
+  const isSell =
+    type === 'disposable' && (!search.direction || search.direction === 'sell'); // sell is the default direction
```

Comment:
I think this is working
```suggestion
  const isSell =
    type === 'disposable' && search.direction !== 'buy';
```

Reply:
You're right, also works!